RealmTasks Apple/RealmTasks iOS/TablePresenter.swift

```diff
@@ -115,9 +115,9 @@ UITableViewDataSource, UITableViewDelegate, UIGestureRecognizerDelegate where Pa
     // MARK: UITableViewDelegate
 
     func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
-        if cellPresenter.currentlyEditingIndexPath?.row == indexPath.row {
-            return floor(cellPresenter.cellHeight(forText: cellPresenter.currentlyEditingCell!.textView.text))
-        }
+//        if cellPresenter.currentlyEditingIndexPath?.row == indexPath.row {
```

Comment:
IIRC it's there to automatically resize the cell if you type enough to span multiple lines while editing. Did you test that?